akka-docs/src/main/paradox/typed/guide/modules.md

```diff
@@ -229,19 +229,6 @@ Streams solve the following challenges:
 * How to connect asynchronous services in a flexible way to each other with high performance.
 * How to provide or consume Reactive Streams compliant interfaces to interface with a third party library.
 
-### HTTP
-
-[Akka HTTP](https://doc.akka.io/docs/akka-http/current/) is a separate module from Akka.
-
-The de facto standard for providing APIs remotely, internal or external, is [HTTP](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol). Akka provides a library to construct or consume such HTTP services by giving a set of tools to create HTTP services (and serve them) and a client that can be
-used to consume other services. These tools are particularly suited to streaming in and out a large set of data or real-time events by leveraging the underlying model of Akka Streams.
-
-Some of the challenges that HTTP tackles:
-
-* How to expose services of a system or cluster to the external world via an HTTP API in a performant way.
-* How to stream large datasets in and out of a system using HTTP.
-* How to stream live events in and out of a system using HTTP.
-
```

Comment:
I'm not convinced this is a good change. For us it might make sense to only list modules in the core repository, but from a user perspective it would be better to include Akka HTTP since that is an essential part of building systems with Akka. I'd even like to expand this list with a few more:
* Akka gRPC
* Akka Projections
* Alpakka

Since this is in the Getting Started Guide it will be read by new users and they don't care which repository the code is located in.

Reply:
I agree here that documentation structure ought to be geared more toward user discovery than code organization.  Each of those modules are massively valuable (gRPC, Projections, and Alpakka) in the Akka ecosystem and it makes sense that they would get first class attention in the Getting Started Guide.  

Am happy to make any corresponding updates, but perhaps will wait for others on the team to chime in and reach a conclusion.

Reply:
@dkichler Please go ahead with those changes. Add back HTTP, and add gRPC, Projections, and Alpakka.

I'm ok with duplicating those also in the external modules section.

Reply:
I've added in the relevant sections for a review.  Happy to squash this once the changes are accepted as well.